charts/linkerd2/templates/hook-pre-install-namespace.yaml

```diff
@@ -11,6 +11,8 @@ metadata:
   name: {{ .Namespace }}
   annotations:
     {{.ProxyInjectAnnotation}}: {{.ProxyInjectDisabled}}
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "-10"
   labels:
     {{.LinkerdNamespaceLabel}}: "true"
     config.linkerd.io/admission-webhooks: disabled
```

Comment:
As you might have noticed our cli tools also rely on the helm chart. So if we shuffle files around, we need to make sure we are not breaking something on the cli. Currently when I compile the cli on your branch i get the error: 

```sh
➜  linkerd2 git:(use-cert-manager) ✗ bin/linkerd install --ignore-cluster
Error: open /linkerd2/templates/namespace.yaml: file does not exist
```